force-app/main/default/lwc/marketingCloudCopadoDataTable/marketingCloudCopadoDataTable.js

```diff
@@ -421,10 +423,16 @@ export default class MarketingCloudCopadoDataTable extends LightningElement {
     this.sortData(this.sortedBy, this.sortDirection);
   }
 
-  sortData(fieldname, direction) {
-    
-    let parseData = JSON.parse(JSON.stringify( this.visibleData));
-    // Return the value stored in the field
+  sortData(fieldname, direction) { 
+    let parseData='';
+    // 68 sort the data 
+    if(this.data){
+      parseData = JSON.parse(JSON.stringify(this.data));
+      this.data=null;
```

Comment:
why reset this.data to null?

Reply:
@anil-sfdc ?

Reply:
not. required to set null. this.data usinging  handleSearch method